daemon/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "daemon"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+anyhow = "1"
+bitcoin = { version = "0.27", features = ["use-serde"] }
```

Comment:
Maybe depend on `bdk` right away? That is what we will need eventually.

In `xmr-btc-swap`, it was a pain to try and unify dependencies from all `bitcoin`, `bdk` and `miniscript`. Would be nice to avoid this here.

Also, dependabot has an easier time to update things if you access everything through `bdk`.

Reply:
Ok! Then I'll switch that for a `bdk` dependency and remove `bitcoin`